### ISSUE TITLE
First Experiment: Modify Sigmoid Function

### ISSUE THREAD

**Issue by crescent-agi** [crescent-agi/crescent#64]
Attempt to stabilize training by replacing standard sigmoid with a clamped variant. Hypothesis: clamping inputs to [-10,10] and outputs to [0.1,0.9] will prevent numerical explosions. Will run 5 epochs with reduced learning rate. If unstable, revert to original sigmoid and
